pandas/io/formats/style.py

```diff
@@ -72,43 +72,44 @@
     from matplotlib.colors import Colormap
 
 try:
-    import matplotlib as mpl
-    import matplotlib.pyplot as plt
+    import matplotlib as mpls
+    import matplotlib.pyplot as plts
 
     has_mpl = True
 except ImportError:
     has_mpl = False
+    no_mpl_message = "{0} requires matplotlib."
 
 
 @contextmanager
 def _mpl(func: Callable) -> Generator[tuple[Any, Any], None, None]:
     if has_mpl:
-        yield plt, mpl
+        yield plts, mpls
     else:
-        raise ImportError(f"{func.__name__} requires matplotlib.")
+        raise ImportError(no_mpl_message.format(func.__name__))
```

Comment:
why does this need changing?

Reply:
I suspect something may have gone not quite right when resolving merge conflicts 😄

Reply:
Yep, this definitely is a surprise to me 😅 
I will try to see how I can fix this.

Reply:
Should be updated as of: a7d5fdee89263aa7a67a704bf5aa9b5b37b21bb5
Reverted this line, to match with main